scripts/upload_processor.py

```diff
@@ -0,0 +1,46 @@
+"""A function to process uploaded samples"""
+
+import json
+import google.auth
+from airtable import Airtable
+from google.cloud import secretmanager
+
+
+def get_table(project_config):
+    """Get Table from Airtable"""
+
+    # Get the Airtable credentials.
+    base_key = project_config.get('baseKey')
+    table_name = project_config.get('tableName')
+    api_key = project_config.get('apiKey')
+
+    # Pull table
+    table = Airtable(base_key, table_name, api_key)
+
+    return table
+
+
+def get_config():
+    """Retrieve configuration file from secret manager"""
+
+    # secret_id set for testing, TODO: Replace
```

Comment:
Try not to leave TODOs in the code. It's sometimes okay, but work towards a state of the code that's always executable, even if it's work in progress.